airbyte-webapp-e2e-tests/cypress.json

```diff
@@ -8,6 +8,8 @@
     "destination.spec.js",
     "source.spec.js"
   ],
+  "viewportHeight": 800, 
+  "viewportWidth": 1280,
```

Comment:
The default for Cypress is 1000px which is really a tablet size... 1280 seems more appropriate for desktop.